### PR TITLE
Add camera

### PR DESCRIPTION
**Pull request by zohaibahmed-22**

### CODE REVIEW

File: ios/WahedMobile/Info.plist [hunk 1]
```diff
@@ -39,6 +39,12 @@
 	</dict>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string></string>
+	<key>NSCameraUsageDescription</key>
```
Comment (jantebeest) [inline ios/WahedMobile/Info.plist]: Let's come up with some proper messages for these and make the microphone and camera usage descriptions match the ones from the Onfido SDK

File: src/screens/onfido/address_proof_screen.tsx [hunk 2]
```diff
@@ -22,45 +42,60 @@ export default class AddressProofScreen extends PureComponent<Props, State> {
...
+        <View>
+          <Text
+            style={{
+              fontSize: 26,
+              fontWeight: '600',
+              color: '#1F2D3D',
+              textAlign: 'center'
+            }}
+          >
+            Slide up to upload your address proof document
```
Comment (jantebeest) [inline src/screens/onfido/address_proof_screen.tsx]: Shouldn't this be "Proof of address document" or something?

Comment (zohaibahmed-22) [reply src/screens/onfido/address_proof_screen.tsx]: updated

File: ios/WahedMobile/Info.plist (hunk 1 above)
Comment (zohaibahmed-22) [reply ios/WahedMobile/Info.plist]: updated

**Pull request merged**